ggshield/cmd/secret/scan/precommit.py

```diff
@@ -40,6 +41,11 @@ def precommit_cmd(
     Scan as a pre-commit hook all changes that have been staged in a git repository.
     """
     config: Config = ctx.obj["config"]
+
+    if check_user_requested_skip():
+        click.echo("Skipping GitGuardian pre-receive hook.", err=False)
```

Comment:
This is pre-commit here, not pre-receive ;). Actually if we make the message more generic (like "Skipping ggshield hook"), we can make `check_user_requested_skip()` print it and avoid duplicating the message in all hooks of all verticals.